packages/kit-headless/src/components/select/select.tsx

```diff
@@ -71,7 +72,11 @@ export const SelectRoot = component$(
       const expanded = track(() => isExpanded.value);
 
       if (expanded && trigger && listBox) {
```

Comment:
Maybe split this up in 2 if-statements.

```suggestion
     if(!trigger || !listBox) return;

      if (expanded === true) {
```

Reply:
There is also an if-Statement below, that could be merged here after the "if-statement-split" is done.

```ts
if (expanded === true) {
  listBox.style.visibility = 'hidden';

  await updatePosition(trigger, listBox);

  listBox.style.visibility = 'visible';

  listBox.focus(); // from the if-statement below
}
```

Reply:
This seems a lot more declarative/explicit. Good catch. Let me make sure the functionality is the same and I'll commit the changes.

Reply:
Yep looks good. Gonna add the new commit